### DISCUSSION TITLE
Move to Next Chapter when Clicking on Long Strip Mode

### DISCUSSION THREAD

**Discussion by Kinasho** [xgi/houdoku#141] [Feature Requests]
When using long strip mode, when you want to move to the next chapter, you either have to use the arrow keys or click on the icon for the next page on the menu bar. From a UI perspective, a mouse user who would prefer a full screen mode may find it annoying to have to use the keyboard or press the shortcut key to show the menu bar, and then click on the icon. 

Instead, I would prefer if there were a feature whereby you can click on the last page of a manga in Long Strip Mode and be able to move on to the next chapter of a manga. (I say last page since people might accidentally click in long strip mode midway through reading, which might create an annoyance.)

**Comment by xgi** [comment]
Ah, for a while there was actually behavior where even in the longstrip mode, clicking the left/right side of a page would scroll to the previous/next page, same as in the other modes. It didn't really make sense, but I used it sometimes -- it must have gotten removed with a bugfix some time ago.

This is a good suggestion, but to make it simple I may just have it so clicking the top ~30% of the screen goes to the previous page and the bottom ~30% goes to the next, changing chapters if necessary. Basically the same as how the single/double mode works, but vertical.

**Reply by Kinasho** [reply]
Actually yeah, that sounds better since well, nobody really moves their mouse while reading, but they might accidentally click. Sounds good to me
